osbs/utils.py

```diff
@@ -554,3 +556,22 @@ def get_name_and_value(self, label_type):
             return self._label_values[label_type]
         else:
             return (label_type, self._df_labels[label_type])
+
+
+def retry_on_conflict(func,
+                      retry_times=OS_CONFLICT_MAX_RETRIES,
+                      retry_delay=OS_CONFLICT_WAIT):
+    @wraps(func)
+    def retry(*args, **kwargs):
+        for counter in range(retry_times + 1):
+            try:
+                return func(*args, **kwargs)
+            except OsbsResponseException as ex:
+                if ex.status_code == 409 and counter != retry_times:
```

Comment:
Use `http_client.CONFLICT` instead of `409`

Reply:
ok